tools/publish.py

```diff
@@ -0,0 +1,151 @@
+from gladier.defaults import GladierDefaults
```

Comment:
Can you break that into __init__ like the other functions?

Reply:
Sure, happy to!